Modules/socketmodule.c

```diff
@@ -5787,9 +5787,14 @@ gethost_common(socket_state *state, struct hostent *h, struct sockaddr *addr,
 
     /* SF #1511317: h_aliases can be NULL */
     if (h->h_aliases) {
-        for (pch = h->h_aliases; *pch != NULL; pch++) {
+        for (pch = h->h_aliases; ; pch++) {
             int status;
-            tmp = PyUnicode_FromString(*pch);
+            char *s;
```

Comment:
rather than `s` for the variable name, lets call it `host_alias` to more clearly indicate what it is.  (here and the loop below)